app/models/spree/order_decorator.rb

```diff
@@ -226,6 +226,10 @@ def set_variant_attributes(variant, attributes)
     end
   end
 
+  def shipment
```

Comment:
this is required for this test:
https://github.com/openfoodfoundation/openfoodnetwork/blob/2-0-stable/spec/models/spree/order_spec.rb#L711